tests/test_waku_discv5.nim

```diff
@@ -320,6 +320,7 @@ procSuite "Waku Discovery v5":
 
       var builder = EnrBuilder.init(enrPrivKey, seqNum = enrSeqNum)
       require builder.withWakuRelaySharding(shardsTopics).isOk()
+      builder.withWakuCapabilities(Relay)
```

Comment:
Just for me to understand better. Is this directly related to the change or why is not needed now?

Reply:
When creating ENRs for tests, without specifying a Capability discv5 will reject it as invalid.

Reply:
ah yes! lovely! makes sense :)